libbeat/scripts/generate_template.py

```diff
@@ -292,12 +293,12 @@ def fill_field_properties(args, field, defaults, path):
             path = field["name"]
         prop, dynamic = fill_section_properties(args, field, defaults, path)
 
+        properties[field.get("name")] = {
+            "type": "nested",
```

Comment:
Strictly speaking it is not needed, but it was there before, so not preferences from my side to keep or remove.